src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
+Interestingly, rendering changed the meta robots tag in 0.16% of pages. While there is no inherent issue with using JavaScript to add a meta robots tag to a page or change its content, SEOs should be judicious in execution. If a page loads with a `noindex` directive in the meta robots tag before rendering, [search engines won't run the JavaScript](https://developers.google.com/search/docs/guides/javascript-seo-basics#use-meta-robots-tags-carefully) that changes the tag value or index the page.
 
 #### Canonicalization
 
-Canonical tags -as described by [Google here](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls)- are used to specify to search engines which is the preferred canonical URL version to index and rank for a page -the one that is considered to be better representative of it- when there are many URLs featuring the same or very similar content. It is important to note that:
+[Canonical tags](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls), as described by Google, are used to specify to search engines which is the preferred canonical URL version to index and rank for a page—the one that is considered to be better representative of it—when there are many URLs featuring the same or very similar content. It is important to note that:
 
 - The canonical tag configuration is used along with other signals to select the canonical URL of a page; it is not the only one.
-- Although self-referencing canonical tags are sometimes used, these aren’t a requirement.
+- Although self-referencing canonical tags are sometimes used, these aren't a requirement.
 
-[Last year](../2019/seo#canonicalization), it was identified that 48.34% of mobile home pages were using a canonical tag. This year the number of mobile home pages featuring a canonical tag has grown to 53.61%.
+{# TODO(authors): The intro disclaims upfront that the dataset is limited to home pages. Consider generalizing to "mobile/desktop pages" throughout the rest of the chapter. #}
+[In last year's chapter](../2019/seo#canonicalization), it was identified that 48.34% of mobile home pages were using a canonical tag. This year the number of mobile home pages featuring a canonical tag has grown to 53.61%.
 
+{# TODO(analysts): It's not clear that B and C add up to A in this chart. It took me a while to realize that they weren't each different types of tags. If the total of 53.61% is significant, I'd recommend a "big number" figure to call attention to it, and only include B and C in this chart. #}
```

Comment:
@Tiggerito

Reply:
@rviscomi  @aleyda 

Almost true. I checked the code and there are pages that can be both a self canonical and a canonical to another URL (0.27%, 0.28%). Stupid thing to do, but means B+C is slightly greater than A.

The related comment covers what I would think is important. The B and C. Maybe word it differently using A and C:

_When analyzing this year's mobile pages canonical tag configuration, it was identified that 48.34% of mobile pages were using a canonical tag and 8.45% of pages had canonical tags pointing to different URLs as the canonical ones._

I also noticed the x-axis label for the chart is wrong for this and a few others. @antoineeripret is that something easy for you to fix?

Reply:
@Tiggerito : I fixed this one and 2 other ones as well. That should be it :) 

Thanks for the heads up.